test/deploy/tools/dex.yaml

```diff
@@ -15,6 +15,8 @@ web:
   https: 0.0.0.0:8443
   tlsCert: /var/certs/dex.pem
   tlsKey: /var/certs/dex-key.pem
+expiry:
+  idTokens: 20m #! this is the lifetime for the id token as well as the access token.
```

Comment:
We should have an integration test that asserts that the pinniped CLI issues a warning with this config + no offline access scope requested.